src/sentry/statistical_detectors/detector.py

```diff
@@ -214,8 +214,8 @@ def detect_regressions(
             request = {
                 "data": data,
                 "sort": "-trend_percentage()",
-                "min_change()": cls.min_change,
-                # "trend_percentage()": 0.5,  # require a minimum 50% increase
```

Comment:
Was this updated in seer? Last I checked, this param required parenthesis.

Reply:
For awhile (since changes I made) it has been a working alias so that both work: https://github.com/getsentry/seer/blob/6f656ea74d695c59b504e5648e615861038e04d9/src/seer/trend_detection/trend_detector.py#L52

This change also pushes us closer to uhm, making this all implementable with gRPC 🤫 https://github.com/getsentry/seer/pull/798